xchange-core/src/main/java/org/knowm/xchange/dto/Order.java

```diff
@@ -86,6 +88,31 @@ public Order(
     this.status = status;
   }
 
+  public Order(
+      OrderType type,
+      BigDecimal originalAmount,
+      CurrencyPair currencyPair,
+      String id,
+      Date timestamp,
+      BigDecimal averagePrice,
+      BigDecimal cumulativeAmount,
+      BigDecimal cumulativeQuoteAmount,
```

Comment:
I don't think we need to store this value is easly calculated: `cumulativeAmount.multiply(averagePrice, ...)`
it would be more universal for all modules to add a convinance method that calculates this on the fly if cumulativeAmount and averagePrice are not null

Reply:
besides that we don't use the term "quote", we call it "counter"

Reply:
@walec51 Thanks for clarifying pushed new changes.